canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.jsx

```diff
@@ -74,27 +78,60 @@ class CheckboxsetControl extends React.Component {
 	}
 
 	render() {
+
+
+		const hidden = this.props.state === STATES.HIDDEN;
+		const disabledTip = this.props.state === STATES.DISABLED;
+
 		let controlValue = this.props.value;
 		if (typeof controlValue === "undefined" || controlValue === null) {
 			controlValue = [];
 		}
 		const checkboxes = [];
 		for (var i = 0; i < this.props.control.values.length; i++) {
+			let tooltip = "";
+			if (this.props.control.description && !this.props.tableControl) {
+				tooltip = (
+					<span >{this.props.control.valueDescs[i]}</span>
+				);
+				// If tooltip has a link, add propertyId in the link object
+				if (this.props.control.description.link) {
```

Comment:
Remove this.  There won't ever be links in these tooltips.  This would only be at the control level but not the value level.

Reply:
Removed it.